tools/tcpstates.py

```diff
@@ -144,8 +144,8 @@
             .oldstate = args->oldstate, .newstate = args->newstate};
         data6.skaddr = (u64)args->skaddr;
         data6.ts_us = bpf_ktime_get_ns() / 1000;
-        bpf_probe_read(&data6.saddr, sizeof(data6.saddr), args->saddr_v6);
-        bpf_probe_read(&data6.daddr, sizeof(data6.daddr), args->saddr_v6);
+        __builtin_memcpy(&data6.saddr, args->saddr, sizeof(data6.saddr));
+        __builtin_memcpy(&data6.daddr, args->daddr, sizeof(data6.daddr));
```

Comment:
Same here: `saddr_v6` and `daddr_v6`.

Reply:
@pchaigno , Paul , is the code looks good to you ?